docker-compose.yml

```diff
@@ -13,6 +13,7 @@ services:
       POSTGRES_USER: ${POSTGRES_USER-postgres}
       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD-postgres}
       POSTGRES_HOST: db
+      SECRET_KEY_BASE: ${SECRET_KEY_BASE-1234567890}
```

Comment:
Hey @santanaluiz, just curious here, are you running the app in production mode using `docker-compose`?

Our compose file is meant to be used for development purposes only, and I don't think the secret_key_base is needed in those cases.

Reply:
I am not using for production, only dev env.
Running rspec fails to me if I don't add it

Reply:
This doesn't sound correct. According to the [Rails docs](https://edgeguides.rubyonrails.org/security.html#session-storage):

> In test and development applications get a secret_key_base derived from the app name

I don't think this is needed to run the tests.

Reply:
This is weird, I cannot reproduce it. I'm actually getting an error but doesn't seem to be related to the secret key

Reply:
@santanaluiz can you please check if this PR fixes your tests https://github.com/rootstrap/rails_api_base/pull/603?